app/controllers/v0/ppiu_controller.rb

```diff
@@ -14,6 +14,8 @@ def update
       pay_info = EVSS::PPIU::PaymentAccount.new(ppiu_params)
       raise Common::Exceptions::ValidationErrors, pay_info unless pay_info.valid?
       response = service.update_payment_information(pay_info)
+      email = current_user.vet360_contact_info&.email || current_user.email
```

Comment:
Any reason not to follow method extraction for readability/code isolation? e.g.,

```rb
before_action :validate_pay_info, only: :update

# ...

def update
  response = service.update_payment_information(pay_info)
  build_and_send_mailer
  render json: response,
         serializer: PPIUSerializer
end

# ...

private

def build_and_send_mailer
  DirectDepositMailer.build(email, params[:ga_client_id]).deliver_now if current_user_email
end

def current_user_email
  current_user.vet360_contact_info&.email || current_user.email
end

def pay_info
  EVSS::PPIU::PaymentAccount.new(ppiu_params)
end

def validate_pay_info
  raise Common::Exceptions::ValidationErrors, pay_info unless pay_info.valid?
end

# ...
```

Reply:
Do we always want to send a mailer? Is there any response validation that might need to apply?

Reply:
@SMLuthi correct me if I'm wrong, but after looking at the specs, it looks like if we get to that line, we've got a successful response.

Reply:
I like that refactor. Except the `build_and_send_mailer` is probably unnecessary.